MeshLib/Elements/MapBulkElementPoint.cpp

```diff
@@ -14,6 +14,25 @@
 
 namespace MeshLib
 {
+MathLib::Point3d getBulkElementPoint(MeshLib::Tri const& /*tri*/,
+                                     std::size_t const face_id,
+                                     MathLib::WeightedPoint1D const& wp)
+{
+    switch (face_id)
+    {
+        case 0:
+            return MathLib::Point3d{std::array<double, 3>{{wp[0], 0.0, 0.0}}};
+        case 1:
+            return MathLib::Point3d{
+                std::array<double, 3>{{1 - wp[0], wp[0], 0.0}}};
+        case 2:
+            return MathLib::Point3d{
+                std::array<double, 3>{{0.0, 1 - wp[0], 0.0}}};
+        default:
+            OGS_FATAL("Invalid face id '%u' for the tri.", face_id);
```

Comment:
```suggestion
            OGS_FATAL("Invalid face id '{:d}' for the tri.", face_id);
```

Reply:
Yes, you are right. Will fix it tomorrow.